servers/src/test/java/tachyon/master/file/meta/AbstractInodeTest.java

```diff
@@ -20,8 +20,6 @@
 
 import tachyon.Constants;
 import tachyon.master.block.BlockId;
-import tachyon.master.file.meta.InodeDirectory;
```

Comment:
ah, technically this import is used ... but it is not needed because the test class is in the same package